src/tests/execute-wp-cli.test.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @jest-environment node
+ */
+
+import getWpNowConfig from '../../vendor/wp-now/src/config';
+import { downloadWPCLI } from '../../vendor/wp-now/src/download';
+import { executeWPCli } from '../../vendor/wp-now/src/execute-wp-cli';
+import getWpCliPath from '../../vendor/wp-now/src/get-wp-cli-path';
+import startWPNow from '../../vendor/wp-now/src/wp-now';
+
+jest.mock( '../../vendor/wp-now/src/download' );
+jest.mock( '../../vendor/wp-now/src/wp-now' );
+jest.mock( '../../vendor/wp-now/src/get-wp-cli-path' );
+jest.mock( '../../vendor/wp-now/src/config' );
+
+const mockPhpInstance = {
+	setSapiName: jest.fn(),
+	mkdir: jest.fn(),
+	writeFile: jest.fn(),
+	mount: jest.fn(),
+	run: jest.fn().mockResolvedValue( {
+		text: 'admin user yoda',
+		errors: 'serious error',
+	} ),
```

Comment:
We need to remove this mocks. This test should execute a real Playground instance. Let's figure it out together tomorrow.

Reply:
Sounds good, I will also take a look on my end before we pair 👍

Reply:
Do you want to update those based on our pairing session?

Reply:
yes!, on it!